api/services/reporting/report_generator.py

```
@@ -0,0 +1,207 @@
+            template: Template file name to use
+
+        Returns:
+            bytes: PDF file as bytes
+        """
+        try:
+            logger.info("Starting PDF generation for audit report")
+
+            # Prepare context data
+            context = {
+                'report': audit_report,
+                'timestamp': datetime.now().strftime("%d/%m/%Y às %H:%M:%S"),
```

Ruff (0.14.5)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)